l10n_es_aeat_mod303/data/2023/aeat.model.export.config.line.csv

```diff
@@ -48,7 +48,7 @@
 "aeat_mod303_2023_sub01_export_line_47","aeat_mod303_2023_sub01_export_config",47,"Liquidación: Regimen General - IVA Devengado - Recargo equivalencia 1.75% - Tipo % [157]","string",5,2,"right",0,"N",0," ","X",,"00175",,,
 "aeat_mod303_2023_sub01_export_line_48","aeat_mod303_2023_sub01_export_config",48,"Liquidación: Regimen General - IVA Devengado - Recargo equivalencia 1.75% - Cuota [158]","float",17,2,"right",0,"N",0," ","X","${object.tax_line_ids.filtered(lambda r: r.field_number==158).amount}",,,,
 "aeat_mod303_2023_sub01_export_line_49","aeat_mod303_2023_sub01_export_config",49,"Liquidación: Regimen General - IVA Devengado - Recargo equivalencia 0%/0.5%/0.62% - Base imponible [16]","float",17,2,"right",0,"N",0," ","X","${object.tax_line_ids.filtered(lambda r: r.field_number==16).amount}",,,,
-"aeat_mod303_2023_sub01_export_line_50","aeat_mod303_2023_sub01_export_config",50,"Liquidación: Regimen General - IVA Devengado - Recargo equivalencia 0%/0.5%/0.62% - Tipo % [17]","float",5,2,"right",0,"N",0," ","X","${object.tax_line_ids.filtered(lambda r: r.field_number==18).move_line_ids.tax_line_id[:1].amount}",,,,
+"aeat_mod303_2023_sub01_export_line_50","aeat_mod303_2023_sub01_export_config",50,"Liquidación: Regimen General - IVA Devengado - Recargo equivalencia 0%/0.5%/0.62% - Tipo % [17]","string",5,2,"right",0,"N",0," ","X",,"00000, 00050, 00062",,,
```

Comment:
se puede dejar como estaba pero con `mapped('move_line_ids.tax_line_id')`, ya que eso es algo que hace internamente en la 13.0 de donde lo arrastré

Reply:
Nosotros hemos decidido hacerlo así para v11, no me parece incorrecto.

Reply:
No sabéis si Hacienda se va a poner pejilguera si declaráis un porcentaje que no es el real.

Reply:
Así he interpretado yo la documentación, que me he podido equivocar.

![image](https://user-images.githubusercontent.com/95038732/220888535-cc0da7e4-5dba-4795-bf3e-3fd30fd6b5ea.png)

Reply:
https://github.com/OCA/l10n-spain/pull/2869#issuecomment-1441563275

Reply:
Ya lo he visto gracias, pues visto así sí, habría que poner el primero por ejemplo, aunque carezca de sentido...